Azure-Samples/azure-search-openai-demo
# Update launch.json with python backend path

## Purpose

This PR adds the python property to the Quart launch.json configuration, so that the debugger selects Python interpreter in the correct virtual environment.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... Please describe:
```

## How to Test

* Without running pip install at the root, try to use the Quart debug configuration. It should work!
